### DISCUSSION TITLE
Question: Temporarily disable VT's link with underlying TRV, to manually set TRV valve to 100%

### DISCUSSION THREAD

**Discussion by zheffie** [jmcollin78/versatile_thermostat#709] [Q&A]
I have a central heating system with a physical living room thermostat that controls a modulating boiler (climate entity in HA). I also have a Sonoff zigbee TRV in my bedroom, which is controlled using VT.

Is it possible to temporarity 'override'  the link of VT to the underlying bedroom TRV in order to full open the TRV valve and let regulation be done by controlling the living room thermostat (which then modulates the boiler heating)?

The way I imagine it, I would need two VT's:
- VT 1 is controlling the living room thermostat
- VT 2 is controlling bedroom TRV

_During Daytime_ => VT 1 is using living room temp and setpoint (21degC) to control living room thermostat. Living room radiators are open 100% and VT 2 is controlling bedroom TRV (15degC)

_Nighttime_ => VT 1 is using bedroom temp and setpoint to control living room thermostat. Bedroom setpoint is 18degC but radiators are open 100% and VT 2 is inactive or temp disabled. (Living room radatiors are closed fully)

off course, necessary automation would be in place to make sure that correct temps and sensors are fed into VT

what I tried already:
- turn off bedroom VT at night, but this also turns off the TRV
- create helper setpoint, and use automations to set thermostats. This works, but increases the complexity of the whole control

**Comment by jmcollin78** [comment]
Hello @zheffie 

Because this is not an issue on VTherm, I will transform your question into a discussion.
